util/storeutil/storeVars.go

```diff
@@ -0,0 +1,8 @@
+package storeutil
```

Comment:
Please rename the filename to `store_vars.go` (it's more idiomatic).

Reply:
Addressed.